include/systems/system.h

```diff
@@ -1573,6 +1573,11 @@ class System : public ReferenceCountedObject<System>,
    */
   std::vector<Number> qoi;
 
+  /**
+   * Vector to hold error estimates for qois
+   */
+  std::vector<Number> qoi_error_estimates;
```

Comment:
Does this belong in the System, like qoi, or as a separate output, like our ErrorVector results?

With System::qoi we have other code that needs to calculate it, so we can get a little savings by keeping it around in a well-defined place to avoid making that calculation redundantly.  Does the same apply here?

Reply:
This isnt going to be a NumericVector elementwise vector like ErrorVector. The global QoI_elementwise_error is a weighted sum of contributions from each QoI, so we dont need to maintain separate vectors for each QoI.